releases/release-1.13/README.md

```diff
@@ -0,0 +1,139 @@
+# Kubernetes 1.13 Release Schedule
+
+*Handy Links*
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.13/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.13/release_team.md)
+* Zoom: join Google Group to receive meeting invite
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Enhancements Tracking Sheet](http://bit.ly/k8s113-enhancements)
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/release.md)
+* [Meeting Minutes](http://bit.ly/k8s113-minutes)
+* [CI Signal Report](http://bit.ly/k8s113-cisignal)
+* [Retrospective Document](http://bit.ly/k8s113-retro)
+
+**tl;dr** The 1.13 release cycle begins on **Monday, October 1st, 2018**, and ends on release day, **Monday, December 03, 2018**.  Enhancements freeze is **Tuesday, October 23, 2018**.  Code slush is **Friday, November 09, 2018**. Code freeze begins **Thursday, November 15, 2018** and ends **Wednesday, November 28, 2018**, but [may vary depending on CI Signal](#conditionally-shortened-code-freeze). Docs must be completed and reviewed by **Tuesday, November 27, 2018**. Release blocking test case additions must be completed by in conjunction with enhancements by code freeze.
+
+## Notes About this Release
+
+* The Release Team will continue with the shortened Code Freeze, [see below](#conditionally-shortened-code-freeze).
+* The feature process is remaining as it has in prior releases, except the it is being renamed from features to Enhancements.
+* Enhancements that don't have complete code and tests by [Code Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.13/release-1.13.md#code-freeze) may be disabled by the release team before cutting the first beta.
+* The release team will escalate [release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [release-master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) failures to SIGs throughout the cycle, not just near release cuts.
+* Unlike the previous cycles key deliverables, including the final release, do **not** fall on Tuesdays due to shortened release cycle.
+* The release cycle length is approximately only **10 weeks** long.
+
+## Timeline
+
+
+| **What** | **Who** | **Oct** | **Nov** | **Dec** | **Jan** | **DEV WEEK** | **TEST GATES** |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Start of Release Cycle | Lead | 1| | | | week 1 | |
+| Finalize Schedule | Lead | 5 | | | | | |
+| Begin collecting planned work from SIGs | Lead, Enhancements Lead | 8| | | | week 2 | |
+| Begin weekly release team meetings | Lead | 8 | | | | | |
```

Comment:
I suggest starting your weeklies on the 1st, especially since there will be more need to coordinate tightly on features^H^H^Henhancements related discussions.

Reply:
done